pkg/controller/cincinnati/cincinnati_controller.go

```diff
@@ -183,6 +187,49 @@ func handleErr(reqLogger logr.Logger, status *cv1alpha1.CincinnatiStatus, reason
 	reqLogger.Error(e, reason)
 }
 
+func (r *ReconcileCincinnati) ensureAdditionalTrustedCA(ctx context.Context, reqLogger logr.Logger, instance *cv1alpha1.Cincinnati, resources *kubeResources) error {
+	// Check if the Cluster is aware of a registry requiring an
+	// AdditionalTrustedCA
+	image := &apicfgv1.Image{}
+	err := r.client.Get(ctx, types.NamespacedName{Name: defaults.ImageConfigName, Namespace: ""}, image)
+	if err != nil {
```

Comment:
Is a "not found" error possible here? Would some clusters just not have this resource?

Reply:
It's possible, but it gets scooped up in the if block.

My thinking here is that if there's any error, back out and ignore any CA request.  Line 199 expands on this by catching the case where the object exists, but there's no `AdditionalTrustedCA`.  The last check is line 205, which makes sure the configmap with the expected name actually exists.   So after line 205, I would expect to always a valid configmap.    

I also don't check for a valid `CertConfigMapKey` because kubernetes will handle that when the mount fails.

Reply:
If there's a connected cluster where this resource just doesn't exist at all, and isn't needed at all, seems like this function should not return an error. Returning an error stops reconciliation and sets an error condition. WDYT?

Reply:
I see what you're saying.  It should returning `nil`.